keyboards/handwired/lagrange/keymaps/default/keymap.c

```diff
@@ -0,0 +1,72 @@
+/* Copyright 2020 Dimitris Papavasiliou <dpapavas@protonmail.ch>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define EQL_ALT MT(MOD_RALT, KC_EQL)
+#define MINS_ALT MT(MOD_LALT, KC_MINS)
+#define HOME_GUI MT(MOD_LGUI, KC_HOME)
+#define RGHT_GUI MT(MOD_RGUI, KC_RGHT)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
```

Comment:
This macro should be re-organized.

Reply:
Assuming you're referring to `LAYOUT`, could you add a bit more detail on how it should be re-organized?

Reply:
Ideally, it should be "side by side" rather than one on the top of the other.